great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -484,8 +484,22 @@ def compute_quantiles(
         axis=0,
         method=quantile_statistic_interpolation_method,
     )
+
+    histogram: Tuple[np.ndarray, np.ndarray] = np.histogram(
+        a=metric_values, bins=NUM_HISTOGRAM_BINS
```

Comment:
Are we planning to allow `NUM_HISTOGRAM_BINS` to be set by the user?

Reply:
@NathanFarmer Right now, there does not seem to be a need for it to be customized.  We can always do it in the future, if called for.  Thanks!